xcodeproj/internal/xcodeproj_rule.bzl

```diff
@@ -220,6 +224,22 @@ targets.
                 additional_linking_files,
             )
 
+        label_to_id[label] = xcode_target.id
+
+        invalid_extra_files_targets = sets.to_list(
+            sets.difference(sets.make(owned_extra_files.values()), targets_labels),
+        )
+        if invalid_extra_files_targets:
+            fail("""\
+Are you using an `alias`? `top_level_target` requires labels of the actual \
+targets: {}
```

Comment:
Comment is out of date now with what field is being used. Should be `associated_extra_files` now.